src/main/java/com/esri/samples/featurelayers/generate_geodatabase/GenerateGeodatabaseSample.java

```diff
@@ -0,0 +1,202 @@
+import javafx.scene.control.Button;
+import javafx.scene.control.ProgressBar;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.concurrent.Job;
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.data.Geodatabase;
+import com.esri.arcgisruntime.data.TileCache;
+import com.esri.arcgisruntime.geometry.Envelope;
+import com.esri.arcgisruntime.layers.ArcGISTiledLayer;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+import com.esri.arcgisruntime.tasks.geodatabase.GenerateGeodatabaseJob;
+import com.esri.arcgisruntime.tasks.geodatabase.GenerateGeodatabaseParameters;
+import com.esri.arcgisruntime.tasks.geodatabase.GeodatabaseSyncTask;
+
+public class GenerateGeodatabaseSample extends Application {
+
+  private MapView mapView;
+  private GenerateGeodatabaseParameters generateGeodatabaseParameters;
+  private AtomicInteger replica = new AtomicInteger();
+
+  private static final String FEATURE_SERVICE_URL =
+      "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Sync/WildfireSync/FeatureServer";
+
+  @Override
+  public void start(Stage stage) throws Exception {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // size the stage, add a title, and set scene to stage
+      stage.setTitle("Generate Geodatabase Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // use a local tile package for the basemap
+      TileCache tileCache = new TileCache("samples-data/sanfrancisco/SanFrancisco.tpk");
+      ArcGISTiledLayer tiledLayer = new ArcGISTiledLayer(tileCache);
+
+      // create a map view and add a map
+      mapView = new MapView();
+      ArcGISMap map = new ArcGISMap(new Basemap(tiledLayer));
+      mapView.setMap(map);
+
+      // create a graphics overlay and symbol to mark the extent
+      GraphicsOverlay graphicsOverlay = new GraphicsOverlay();
+      mapView.getGraphicsOverlays().add(graphicsOverlay);
+      SimpleLineSymbol boundarySymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0xFFFF0000, 5);
+
+      // add a button to generate the geodatabase and a progress bar
+      Button generateButton = new Button("Generate Geodatabase");
+      generateButton.setDisable(true); //wait until sync task loaded
+      ProgressBar progressBar = new ProgressBar();
+      progressBar.setProgress(0.0);
+
+      // create a geodatabase sync task
+      GeodatabaseSyncTask syncTask = new GeodatabaseSyncTask(FEATURE_SERVICE_URL);
```

Comment:
Can just place the URL right above here.
`String featureServiceURL =
          "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Sync/WildfireSync/FeatureServer";`
I know that in other sample we always put it at the top, not really for sure why this is, but I feel like putting it here is easier for the use to reference.